flink/pom.xml

```diff
@@ -34,15 +34,7 @@
             </activation>
             <properties>
                 <flink.scope>compile</flink.scope>
-                <dep.mysql.jdbc.version>8.0.21</dep.mysql.jdbc.version>
             </properties>
-            <dependencies>
-                <dependency>
```

Comment:
Changing from `compile` to `provided` scope is not compatible.
Users should know it.